### PR TITLE
Account for empty comment

### PR DESCRIPTION
**Pull request by daddykotex**

### CODE REVIEW

File: modules/formatter/src/writer/ShapeWriter.scala [hunk 1]
```diff
@@ -285,7 +285,7 @@ object ShapeWriter {
           members
         ) =>
       val content =
-        if (members.members.isEmpty) "{}"
+        if (members.members.isEmpty && !Comment.hasComment(members.ws0)) "{}"
         else s"{\n${indent(members.write, "\n", 4)}\n}"
```
Comment (daddykotex) [inline modules/formatter/src/writer/ShapeWriter.scala:289]: the whitespace writing is already taken care of in `members.write`

File: modules/formatter/src/writer/ShapeWriter.scala [hunk 2]
```diff
@@ -285,7 +285,7 @@ object ShapeWriter {
           members
         ) =>
       val content =
-        if (members.members.isEmpty) "{}"
+        if (members.members.isEmpty && !Comment.hasComment(members.ws0)) "{}"
```
Comment (kubukoz) [inline modules/formatter/src/writer/ShapeWriter.scala:288]: **question:** is this also handled in anonymous inputs/outputs?

Comment (daddykotex) [reply modules/formatter/src/writer/ShapeWriter.scala:288]: line 186 would confirm this:
```
      val content =
        if (members.members.isEmpty && !Comment.hasComment(members.ws0))
          "{}"
        else {
          val indented = indent(members.write, "\n", 4)
          s"{\n${indented}\n}"
        }
```

Comment (kubukoz) [reply modules/formatter/src/writer/ShapeWriter.scala:288]: oh great, thanks

**Review by kubukoz — APPROVED** [review]

**Pull request merged**